scripts/init-submodules.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+git submodule init "$@"
+git submodule update --recursive
```

Comment:
Isn't this enough?

```suggestion
git submodule update --init --recursive
```

Reply:
oh, you're trying to init only the ones we're building? Isn't the update --recursive going to do that anyway?

Reply:
nope, the `--init` there would do it for all submodules in the repo. 

I mean this is a small change but I've seen submodule checkout took some time and there are only going to be more submodules, not less so better start checking out only  what's needed.

That's my overall approach when doing CI - don't try to do everything, just what is really necessary to achieve the step

Reply:
To explain: `init` enables the module (in .git/config) and `update` actually checks out the contents on enable modules